src/scf-release/jobs/acceptance-tests-brain/templates/run.erb

```diff
@@ -49,7 +49,7 @@ for i in "${ASSETS}/plugins/"*; do
 done
 shopt -u nullglob
 
-PARAM="--timeout 600 --verbose"
+PARAM="--timeout 600 --verbose --in-order"
```

Comment:
We probably shouldn't trigger `--in-order` by default. After all, we wrote the test brain runner, and intentionally randomized the test order to catch issues with test dependencies…